pyproject.toml

```diff
@@ -82,7 +85,8 @@ nmslib-metabrainz = {version = "^2.1.3", python = ">=3.11, <3.13", optional = tr
 # The latest torch version available for MacOSX + x86_64 is 2.2.2
 torch = [
     {version = ">=1.6.0, <2.3.0", markers = "sys_platform == 'darwin' and platform_machine == 'x86_64'", optional = true},
-    {version = ">=1.6.0, <3.0.0", optional = true}
+    {version = ">=1.6.0, <2.5.0", python = "3.8", optional = true}, # 2.5.1 is not compatible with Python 3.8
```

Comment:
This is because https://github.com/pytorch/pytorch/issues/139005

Once https://github.com/MobileTeleSystems/RecTools/pull/222 is merged, we can remove this line.